js/lib/tree.js

```diff
@@ -106,12 +99,11 @@ var NodeView = widgets.WidgetView.extend({
             .find('i.jstree-icon.jstree-ocl').first();
     },
 
-    setOpenCloseIcon: function() {
-        var open_icon = this.getOpenIcon();
+    setOpenCloseIcon: function(recursive=false) {
+				var open_icon = this.getOpenIcon();
```

Comment:
Do you mind replacing tabs by spaces everywhere? So we don't have a mix of tabs and spaces